tqdm/_tqdm.py

```diff
@@ -282,6 +285,23 @@ def _decr_instances(cls, instance):
         except KeyError:
             pass
 
+    @classmethod
+    def write(cls, s, file=sys.stderr, end="\n"):
+        """
+        Print a message via tqdm (without overlap with bars)
+        """
+        # Clear all bars
+        for inst in cls._instances:
+            if inst.fp == file:
```

Comment:
This is what caused the bug. In fact, the `.write()` method wasn't working at all, simply because the default `file` for `.write()` was set to `sys.stdout` while `tqdm` progress bar is set to `file=sys.stderr` by default.
In the past it worked, because `.write()` would work on all bars whatever the file. I figured  `sys.stdout` was the most reasonable choice for most people, they don't want to print to `sys.stderr`, but well, I reverted to agree with `tqdm`.